docker/addons/cassandra-writer/docker-compose.yml

```diff
@@ -40,7 +40,6 @@ services:
       MF_CASSANDRA_WRITER_DB_PORT: ${MF_CASSANDRA_WRITER_DB_PORT}
       MF_CASSANDRA_WRITER_DB_CLUSTER: ${MF_CASSANDRA_WRITER_DB_CLUSTER}
       MF_CASSANDRA_WRITER_DB_KEYSPACE: ${MF_CASSANDRA_WRITER_DB_KEYSPACE}
-      MF_CASSANDRA_WRITER_TRANSFORMER: ${MF_CASSANDRA_WRITER_TRANSFORMER}
```

Comment:
These look like strayed changes. Please don't change Compose files of the Writers in this pull request.